### DISCUSSION TITLE
People filter does not show the applied filter value

### DISCUSSION THREAD

**Discussion by chetan-d-mahajan** [microsoft-search/pnp-modern-search#2550] [Q&A]
**Version used**
4.7.0.0

**Describe the bug**
When People filter is applied, the applied value is not shown indicating which filter value is applied.

**To Reproduce**
![image](https://user-images.githubusercontent.com/26963448/201598543-1b1e20b5-3322-4776-a04f-d80353416e4b.png)

**Expected behavior**
Applied people filter value should have been shown on the results page indicating what the user has filtered.

**Screenshots**
Attached above

**Desktop (please complete the following information):**
 - Browser : Chrome, Edge

**Additional context**
Applicable for people results

**Comment by wobba** [comment]
You don't share your exact setup, and what properties you use. If It's "OfficeNumber" you use, it's not refinable so expected. Or is it listing values, and when you apply it don't work? Then you need to share your exact setup for repro.

**Reply by chetan-d-mahajan** [reply]
Thanks for your kind reply!
I had to turn on the "showSelectedFilters" property for the results webpart which fixed  the issue.
